bank2/main.go

```diff
@@ -0,0 +1,267 @@
+// Copyright 2016 The Cockroach Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+// implied. See the License for the specific language governing
+// permissions and limitations under the License. See the AUTHORS file
+// for names of contributors.
+//
+// Author: Spencer Kimball (spencer@cockroachlabs.com)
+
+package main
+
+import (
+	"database/sql"
+	"flag"
+	"fmt"
+	"log"
+	"math/rand"
+	"net/url"
+	"os"
+	"sync/atomic"
+	"time"
+
+	// Import postgres driver.
+	"github.com/cockroachdb/cockroach-go/crdb"
+	_ "github.com/lib/pq"
+)
+
+const systemAccountID = 0
+const initialSystemBalance = 1000000
+const initialBalance = 1000
+
+var maxTransfer = flag.Int("max-transfer", 100, "Maximum amount to transfer in one transaction.")
+var numTransfers = flag.Int("num-transfers", 0, "Number of transfers (0 to continue indefinitely).")
+var numAccounts = flag.Int("num-accounts", 100, "Number of accounts.")
+var concurrency = flag.Int("concurrency", 16, "Number of concurrent actors moving money.")
+var contention = flag.String("contention", "low", "Contention model {low | high}.")
+var balanceCheckInterval = flag.Duration("balance-check-interval", 1*time.Second, "Interval of balance check.")
+
+var txnCount int32
+var successCount int32
+
+type measurement struct {
+	read, write, total time.Duration
+}
+
+func transfersComplete() bool {
+	return *numTransfers > 0 && atomic.AddInt32(&successCount, 1) >= int32(*numTransfers)
```

Comment:
`s/AddInt32/LoadInt32/g`. And then you'll need to increment `successCount` on a successful transfer.